cluster-up/cluster/images.sh

```diff
@@ -17,6 +17,7 @@ if [ -z $KUBEVIRTCI_PROVISION_CHECK ]; then
     IMAGES[okd-4.2]="okd-4.2@sha256:a830064ca7bf5c5c2f15df180f816534e669a9a038fef4919116d61eb33e84c5"
     IMAGES[okd-4.3]="okd-4.3@sha256:63abc3884002a615712dfac5f42785be864ea62006892bf8a086ccdbca8b3d38"
     IMAGES[ocp-4.3]="ocp-4.3@sha256:03a8c736263493961f198b5cb214d9b1fc265ece233c60bdb1c8b8b4b779ee1e"
+    IMAGES[ocp-4.4]="ocp-4.4@sha256:b235e87323ed88c46fedf27e9115573b92f228a82559ab7523dd1be183f66af8"
```

Comment:
how can we verify that the image we have built is based on the slightly-more stable images pointed from https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/ ?

Random images typically die after a few days. Linked images would survive months. Or so I'm told.